engine/src/main/scala/cromwell/engine/workflow/lifecycle/deletion/DeleteWorkflowFilesActor.scala

```diff
@@ -162,7 +163,7 @@ class DeleteWorkflowFilesActor(rootWorkflowId: RootWorkflowId,
       stay
     case Event(ShutdownCommand, _) => stopSelf()
     case other =>
-      log.error(s"Programmer Error: Unexpected message to ${getClass.getSimpleName} ${self.path.name} in state $stateName with $stateData: ${other.toString.take(300)}")
+      log.error(s"Programmer Error: Unexpected message to ${getClass.getSimpleName} ${self.path.name} in state $stateName with $stateData: ${other.toString.elided(300)}")
```

Comment:
yes this was my suggestion but it didn't address @kshakir's more important point about the expense of the `toString` computation 🙂

Reply:
Yup, it just took me a little longer to get that one fully implemented